localization/src/utils/getters.ts

```diff
@@ -52,9 +52,15 @@ export function getDateFormatInfo(): DateFormatInfo {
     localePattern = `${localePattern}${symbols[components[1]]}`;
     localePattern = `${localePattern}${separator}${symbols[components[2]]}`;
 
+    // Determining the first day of the week by locale when using library can consume too much space,
+    // but without it it will be inaccurate,
+    // so for simplicity we will define that on the web all weeks start on Monday, as the most common day.
```

Comment:
I'm not sure if this is very correct behaviour.

I have one suggestion for that which @savelichalex might not like, but it can by applied only for web ;)

1) We can move this file out of `core` to the `localization` package:
https://github.com/tonlabs/UIKit/blob/development/packages/core/src/configs/DayJSConfig.js
2) And extend the `dayjs` by weekday plugin only for web (in order to use it here as `dayjs().weekday()`:
https://day.js.org/docs/en/plugin/weekday

P.S. **Later** we might want to clean up this part in order to avoid using all these plugins on mobile as we unfortunately do right now. And leave them only for web. @savelichalex FYI

Reply:
All that fight for bundle size was useless...